lib/helper/Playwright.js

```diff
@@ -2840,10 +2865,22 @@ class Playwright extends Helper {
 
   /**
    * Returns full URL of request matching parameter "urlMatch".
+   *
+   * Examples:
+   *
+   * ```js
+   * await I.getTrafficUrl('https://api.example.com/session');
+   * await I.getTrafficUrl(/session.*start/);
```

Comment:
I just thought, maybe we should use `grab` prefix? 
juist to keep one standard

Reply:
I agree with DavertMilk.
I also would prefer "grabTrafficUrl".